tests/unit_tests/device/hip/detector_hip_kernel.hip

```diff
@@ -0,0 +1,121 @@
+/** TRACCC library, part of the ACTS project (R&D line)
+ *
+ * (c) 2021-2022 CERN for the benefit of the ACTS project
+ *
+ * Mozilla Public License Version 2.0
+ */
+// hip includes 
+#include <cstdio>  // For printf
+
+#include <hip/hip_runtime.h>
+#include <iostream>
+#include <hip/hip_runtime_api.h>
+
+#include "detray/definitions/detail/hip_definitions.hpp"  
```

Comment:
If the detray `hip_definitions.hpp` is included here, do we still need to include `<hip/hip_runtime_api.h>` and `<hip/hip_runtime.h>` again, or can they be remove?